src/app/_components/EventLinkModal.js

```diff
@@ -1,13 +1,55 @@
 import PropTypes from 'prop-types';
 import { QRCodeSVG } from 'qrcode.react';
+import Instagram from '@icons/instagram.svg';
+import Facebook from '@icons/facebook.svg';
```

Comment:
🟢  You can move these into the Icons index.js file, and import from there

Reply:
thanks, done